src/app/modules/shared_models/active_section.nim

```diff
@@ -127,4 +127,13 @@ QtObject:
     read = nbMembers
 
   proc hasMember(self: ActiveSection, pubkey: string): bool {.slot.} =
-    return self.item.hasMember(pubkey)
\ No newline at end of file
+    return self.item.hasMember(pubkey)
+
+  proc pendingRequestsToJoin(self: ActiveSection): QVariant {.slot.} =
+    if (self.item.id == ""):
+      # FIXME (Jo) I don't know why but the Item is sometimes empty and doing anything here crashes the app
```

Comment:
It's pretty easy. To reproduce, you just need to comment those lines and have a joinedCommunity, for example, create a new community, close the app and reopen

Reply:
@jrainville yes, I managed to reproduce a crash when I removed that "if" condition, thanks for the tip.

What I see... the issue is somehow in relation to my concerns added [here](https://github.com/status-im/status-desktop/pull/4465#pullrequestreview-853734668), there are places on the qml side where you're accessing `pendingRequestsToJoin` and `members` properties as a part of `communitiesModule` (via `observedCommunity`) when we load qml, but you don't have `observedCommunity` set in that moment since it's set from the qml side onlly, on demand via `setObservedCommunity` proc, and that's why you faced a crash.

For the difference how is that working in `mainModule` in `load` we have:
```
# Set active section on app start
self.setActiveSection(activeSection)
```
what ensured that active section is set before qml is loaded, and from that point on we are able to change active section from the qml side.

The thing here is a "mixture" I mentioned above, cause you cannot set "observedCommunity" community before we actually need to do that (so that cannot be set on the app load), but actually `CommunityProfilePopup` and components beneath should operate with appropriate chat section module which represents active community section.

Reply:
I added a new PR on top of this one that removes the use of observedCommunity. Sadly, it didn't fix the crash. You can see the PR here: https://github.com/status-im/status-desktop/pull/4488

Let me know @saledjenic if you have other ideas of how to fix that.